### PR TITLE
feat(bctheme): BCTHEME-51 Optional Custom Options (Modifiers)

### PR DESCRIPTION
**Pull request by BC-tymurbiedukhin**
#### What?

For Custom Options (modifiers) that are not required for the shopper to fill out add the word "Optional" next to the field.

#### Tickets / Documentation

https://jira.bigcommerce.com/browse/BCTHEME-51

#### Screenshots (if appropriate)

Screenshot 2020-07-01 at 19 37 02

ping @yurytut1993 @golcinho

### CODE REVIEW

File: templates/components/common/requireness-msg.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<small>
+    {{#if required}}{{lang 'common.required'}}{{else}}{{lang 'common.optional'}}{{/if}}
+</small>
```
Comment (junedkazi) [inline templates/components/common/requireness-msg.html:3]: Can we add an End of Line here.

File: templates/components/common/requireness-msg.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<small>
+    {{#if required}}{{lang 'common.required'}}{{else}}{{lang 'common.optional'}}{{/if}}
```
Comment (junedkazi) [inline templates/components/common/requireness-msg.html:2]: @bc-as are we sure we want to surface `Optional` as a flag across all not not required fields.

Comment (bc-as) [reply templates/components/common/requireness-msg.html:2]: @junedkazi Yes, let's add it.  We can re-evaluate this into a page builder toggle if necessary

File: templates/components/common/requireness-msg.html (hunk 1 above)
Comment (BC-tymurbiedukhin) [reply templates/components/common/requireness-msg.html]: End of the line was added

File: templates/components/common/requireness-msg.html (hunk 2 above)
Comment (junedkazi) [reply templates/components/common/requireness-msg.html:2]: @bc-as can we create a ticket to explore the `Optional` being `Optional` via Page Builder.

**Pull request merged**